.env.development

```diff
@@ -1,7 +1,7 @@
 REACT_APP_WEBMAP_DOMAIN=http://dev.treetracker.org
 REACT_APP_API_ROOT=https://dev-k8s.treetracker.org/api/admin
 REACT_APP_EARNINGS_ROOT=https://dev-k8s.treetracker.org/earnings
-REACT_APP_MESSAGING_ROOT=https://dev-k8s.treetracker.org/messaging
+REACT_APP_MESSAGING_ROOT=http://localhost:3004
 REACT_APP_QUERY_API_ROOT=https://dev-k8s.treetracker.org/query
```

Comment:
Don't change the defaults here!  You can run locally by making your own local .env file and starting with NODE_ENV= that environment.

Reply:
Nick caught me on the same -- Will do from here on